StackExchange.Redis/StackExchange/Redis/IDatabase.cs

```diff
@@ -45,6 +45,44 @@ public interface IDatabase : IRedis, IDatabaseAsync
         RedisValue DebugObject(RedisKey key, CommandFlags flags = CommandFlags.None);
 
         /// <summary>
+        /// Add the specified member to the set stored at key. Specified members that are already a member of this set are ignored. If key does not exist, a new set is created before adding the specified members.
+        /// </summary>
+        /// <returns>True if the specified member was not already present in the set, else False</returns>
+        /// <remarks>http://redis.io/commands/sadd</remarks>
+        bool GeoAdd(RedisKey key, RedisValue value0, RedisValue value1, RedisValue value2, CommandFlags flags = CommandFlags.None);
+
+        /// <summary>
+        /// Add the specified members to the set stored at key. Specified members that are already a member of this set are ignored. If key does not exist, a new set is created before adding the specified members.
+        /// </summary>
+        /// <returns>the number of elements that were added to the set, not including all the elements already present into the set.</returns>
+        /// <remarks>http://redis.io/commands/geoadd</remarks>
+        long GeoAdd(RedisKey key, RedisValue[] values, CommandFlags flags = CommandFlags.None);
+
+        /// <summary>
+        /// Return the distance between two members in the geospatial index represented by the sorted set.
+        /// </summary>
+        /// <returns>The command returns the distance as a double (represented as a string) in the specified unit, or NULL if one or both the elements are missing.</returns>
+        /// <remarks>http://redis.io/commands/geodist</remarks>
+        RedisValue GeoDist(RedisKey key, RedisValue value0,RedisValue value1,GeoUnit geoUnit = GeoUnit.m, CommandFlags flags = CommandFlags.None);

```

Comment:
I was going to use string, b/c redis returns a double represented by a string, and let the end user decide how to cast it. I used RedisValue b/c it was convenient to use as a string. But, I'm good with making it a double.